assets/dev/js/editor/regions/navigator/navigator.js

```diff
@@ -11,6 +11,7 @@ export default class extends BaseRegion {
 		this.component = $e.components.register( new Component( { manager: this } ) );
 
 		this.isDocked = false;
+		this.storage.size.width = this.$el.css( 'width' );
```

Comment:
If a user has already changed the navigator width in the previous session, this fix will override it

Reply:
Fixed.